frontend/scss/app/components/_task_status.scss

```
@@ -0,0 +1,65 @@
+// Task-state pill, shared by the downloads view and the DVR library.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)